cmd/gcp-controller-manager/main.go

```diff
@@ -72,6 +72,7 @@ var (
 	csrApproverAllowLegacyKubelet      = pflag.Bool("csr-allow-legacy-kubelet", true, "Allow legacy kubelet bootstrap flow.")
 	gceAPIEndpointOverride             = pflag.String("gce-api-endpoint-override", "", "If set, talks to a different GCE API Endpoint. By default it talks to https://www.googleapis.com/compute/v1/projects/")
 	directPath                         = pflag.Bool("direct-path", false, "Enable Direct Path.")
+	enableServiceAccountVerifier       = pflag.Bool("enable-service-account-verifier", false, "Enable service account verifier.")
```

Comment:
Would something like `direct-path-without-channel-authn` make more sense?  IMO "service-account-verifier" is not only very internal  but also can be alarming if users found it being set to false.

Reply:
Good point. Changed to "direct-path-without-channel-authn"